rfd/0170-package-distribution-v3.md

```diff
@@ -0,0 +1,450 @@
+---
+authors: Fred Heinecke (fred.heinecke@goteleport.com)
+state: draft
+---
+
+# RFD 170 - Package Distribution v3
+
+## Required Approvers
+
+* Engineering: @r0mant
+* Product: @klizhentas
+
+## What
+
+Hosting and configuration of APT, YUM, and Zypper repositories, and publishing
+to them. Other binary artifact repos (release server, Helm, Terraform), are
+considered.
+
+## Why
+
+RFD 58 defined our current OS package repo solution. The solution built based
+off of this does not meet our current needs, which were primarily missed
+because:
+* When I wrote the RFD I was new to Teleport, and did not fully understand the
+  impact of the work, and
+* The requirements for this project were not fully captured, and
```

Comment:
I don't think these 2 are relevant to the RFD, I would remove them. It basically reads as "we messed up" :) I would just say that the current solution does not meet our needs and go into issues description below.